api/src/functions/auth.js

```diff
@@ -18,7 +19,13 @@ export const handler = async (event, context) => {
     // address in a toast message so the user will know it worked and where
     // to look for the email.
     handler: (user) => {
-      return user
+      const res = sendEmail({
+        to: user.email,
+        subject: 'Reset SuperStore Password',
```

Comment:
Right now all of our branding is "Superstore". We can change it to "SuperStore", but just suggesting the change for consistency
```suggestion
        subject: 'Reset Superstore Password',
```